### PR TITLE
Feat trusted

### LINKED ISSUE #244: Trusted list
Now we have an admin and non-admin users in the chat. I'd like to add one more level. Let's introduce `Trusted` level.
The idea is pretty clear:
- [x] add `trusted` collection into DB (who trusted, trusted by whom, when trusted);
```json
[
    {
        "userId": 1234567890,
        "trustedBy": 1234,
        "trustedAt": "2022-22-02 10:20"
    }
]
```
- [x] add commands to `trust` and `untrust` for admin (maybe cmd to check is the particular user trusted);
- [x] put or delete trusted users into \ from DB by commands above, (admins are trusted by default);

Why? To allow some non-admins to run some flood dangerous commands, for example #243 
And this badge could be a pleasure for active chat members who are not win a AoC yet ;)

For example:

```
Alice: /cancel
Nyan: *ignore*

admin: /trust @Alice
Nyan: Alice is trused now!

Alice: /cancel
Nyan: *start a Poll*

admin: /untrust @Alice
Nyan: @Alice did not justify the trust...

admin: /trust? @Alice
Nyan: nop
```

I see all this stuff like a part of core functionality.

### PR DESCRIPTION
**Pull request by egregors**
This PR is about of resolve #244 issue

### CODE REVIEW

**Comment by getjump** [comment]
Awesome work as always Mr. Vadim, can you take a look at my proposals?

**Comment by egregors** [comment]
@getjump 

> Awesome work as always Mr. Vadim

Thanks!

> can you take a look at my proposals?

Sure! Which one? About extracting a part with `CommandHandler` handler creating?

File: bot/skills/towel_mode.py [hunk 1]
```diff
@@ -218,6 +218,7 @@ def i_am_a_bot_btn(update: Update, context: CallbackContext):
 
 
 def ban_user(context: CallbackContext):
+    # fixme: smth wrong here
```
Comment (C-Pro) [inline bot/skills/towel_mode.py:221]: exhaustive description

File: bot/skills/trusted_mode.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+    if user and admin and chat_id:
+        if _db.get_user(user.id) is not None:
+            msg = f"{user.name} is already trusted 😼👍"
+        else:
+            _db.trust(user.id, admin.id)
+            msg = f"{user.name} is trusted now! 😼🤝😐"
+
+        answer = context.bot.send_message(chat_id, msg)
+        cleanup_queue_update(
```
Comment (C-Pro) [inline bot/skills/trusted_mode.py:89]: I think it can be useful to keep this instead of cleaning up.

File: bot/skills/trusted_mode.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        )
+
+
+def untrust_callback(update: Update, context: CallbackContext):
+    user, chat_id, _ = _get_user_and_admin(update)
+
+    if user and chat_id:
+        _db.untrust(user.id)
+        answer = context.bot.send_message(chat_id, f"{user.name} lost confidence... 😼🖕")
+        cleanup_queue_update(
```
Comment (C-Pro) [inline bot/skills/trusted_mode.py:103]: same thing here. it is not very frequent operation so it won't spam chat, but it is useful to keep this as a reference

**Review by C-Pro — APPROVED** [review]
LGTM 👍

File: bot/skills/trusted_mode.py (hunk 2 above)
Comment (egregors) [reply bot/skills/trusted_mode.py:89]: Sounds reasonable, thanks. I'll fix it

File: bot/skills/trusted_mode.py (hunk 3 above)
Comment (egregors) [reply bot/skills/trusted_mode.py:103]: Gonna fix it, ty

**Pull request merged**